src/hotspot/cpu/aarch64/frame_aarch64.cpp

```diff
@@ -836,20 +832,15 @@ frame::frame(void* sp, void* fp, void* pc) {
 
 #endif
 
-void JavaFrameAnchor::make_walkable(JavaThread* thread) {
+void JavaFrameAnchor::make_walkable(Thread* thread) {
```

Comment:
No need to change the type of `thread`  (the later cast in the assert was unnecessary). 

Though why is the thread passed in as it is unused except in the assert??

Reply:
> Though why is the thread passed in as it is unused except in the assert??

I don't know. Removing it would clean up the code.

Reply:
Right, I also noticed that. Unneeded `Thread::current()` calls are not nice, either. If we can live without the assertion I'd like to see the thread parameter go away.

Reply:
I removed the assertion.